src/fetch_modules.js

```diff
@@ -1,16 +1,17 @@
-export default function(nodeVersion) {
+export default function (nodeVersion) {
   if (window.wtModuleList) {
-    return $.Deferred().resolve(window.wtModuleList)
+    return $.Deferred().resolve(window.wtModuleList);
   }
 
-  const defaultNodeVersion = 12;
+  const defaultNodeVersion = 16;
   const nodeUrls = {
-    "8": "https://auth0-internal.us8.webtask.io/canirequire",
-    "12": "https://auth0-internal.us12.webtask.io/canirequire",
+    8: "https://auth0-internal.us8.webtask.io/canirequire",
+    12: "https://auth0-internal.us12.webtask.io/canirequire",
+    16: "https://auth0-internal.us12.webtask.io/canirequire",
```

Comment:
We'll need to change this to something else, or specify the header when we call this.